src/public/css/home.css

```diff
@@ -62,7 +62,6 @@ ul {
   display: flex;
   justify-content: space-around;
   align-items: center;
-  flex-wrap: wrap;
   margin-top: 50px;
 }
```

Comment:
Could you add  the same right and left margins to overview also. All sections look aligned now beside overview section.